generate.pl

```diff
@@ -162,16 +172,25 @@ sub die_with_sample {
       $output =~ s/\{\{args\}\}/$builds{$build}/mg;
 
       if ($build =~ /slim/) {
-        $output =~ s/\{\{docker_slim_run_install\}\}/$docker_slim_run_install/mg;
-        $output =~ s/\{\{docker_slim_run_purge\}\}/$docker_slim_run_purge/mg;
+        $output =~ s/\{\{docker_run_install\}\}/$docker_slim_run_install/mg;
+        $output =~ s/\{\{docker_run_purge\}\}/$docker_slim_run_purge/mg;
         $output =~ s/\{\{tag\}\}/$debian_release-slim/mg;
       }
+      elsif ($build =~ /debug/) {
+        $output =~ s/\{\{docker_run_install\}\}/$docker_debug_run_install/mg;
+        # leading space important to reduce diffs against non-debug Dockerfiles
```

Comment:
Just a question: Does this still matter, since there are now just three Dockerfiles which are supported and all others are in EOL?

Reply:
Yeah, need to revisit this

Reply:
Thanks, removed in the latest rebase 🙏